PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 _Describe the purpose of the pull request_
```

Comment:
Not relevant to your fix, but can you rewrite the intro section (first two lines are basically saying the same thing). 

I would remove header and sub-header and replace with this instead

```
- Describe the feature, bug-fix or enhancement
- Add screenshots & information that can help with the review
```

Reply:
Add that as a suggestion (+/-) and I will apply it 😉

Reply:
I cannot access the line above so I'll create a new PR :)